pkg/model/awsmodel/context.go

```diff
@@ -28,5 +29,15 @@ type AWSModelContext struct {
 
 // UseLaunchTemplate checks if we need to use a launch template rather than configuration
 func UseLaunchTemplate(ig *kops.InstanceGroup) bool {
-	return ig.Spec.MixedInstancesPolicy != nil
+	if featureflag.EnableLaunchTemplates.Enabled() {
+		return true
+	}
+	// @note: this mixed instance polices was added before the feature flag, to keep the
```

Comment:
Highlighting the question?

Reply:
Yes, technically we can remove the MixedInstancesPolicy if we do it now, as it hasn't been in a "real" release